components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -373,6 +374,21 @@ public void handle(HttpServletRequest request, HttpServletResponse response,
                     }
                 }
 
+                // Add Diagnostic logs with the filtered authenticator list information.
+                if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                    DiagnosticLog.DiagnosticLogBuilder diagLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                            FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK,
+                            FrameworkConstants.LogConstants.ActionIDs.HANDLE_AUTH_REQUEST);
+                    diagLogBuilder.inputParam(LogConstants.InputKeys.SERVICE_PROVIDER, context.getServiceProviderName())
```

Comment:
check previous comment, using different keys for application

Reply:
This will be fixed with https://github.com/wso2/carbon-identity-framework/pull/4809